starter-code/pizza.js

```diff
@@ -1 +1,112 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$(document).ready(function() {
+
+  // PRICE
+  var startingPrice = 10;
+
+  var pepperonniPrice =1;
+  var mushroomPrice =1;
+  var pepperPrice =1;
+  var saucePrice =0;
+  var crustPrice =0;
```

Comment:
The pattern you used is unexpected. You update the price of each topping & extra depending on whether it is selected or not. Then you add all those prices.

A more sensible way to do it would do not change the price of things but selectively add them. A perk of doing this is to not have to hardcode the price of things into your code. For example if the price of the pepperoni changes, you have to update your code in 2 places (initial value and handler) instead of just one (just the set price).